assets/base.css

```diff
@@ -1070,16 +1064,41 @@ summary::-webkit-details-marker {
 }
 
 @media screen and (min-width: 990px) {
+  .grid--6-col-desktop .grid__item {
+    width: calc(16.66% - var(--grid-desktop-horizontal-spacing) * 5 / 6);
+    max-width: calc(16.66% - var(--grid-desktop-horizontal-spacing) * 5 / 6);
+  }
+
+  .grid--5-col-desktop .grid__item {
+    width: calc(20% - var(--grid-desktop-horizontal-spacing) * 4 / 5);
+    max-width: calc(20% - var(--grid-desktop-horizontal-spacing) * 4 / 5);
+  }
+
   .grid--4-col-desktop .grid__item {
     width: calc(25% - var(--grid-desktop-horizontal-spacing) * 3 / 4);
+    max-width: calc(25% - var(--grid-desktop-horizontal-spacing) * 3 / 4);
   }
 
   .grid--3-col-desktop .grid__item {
     width: calc(33.33% - var(--grid-desktop-horizontal-spacing) * 2 / 3);
+    max-width: calc(33.33% - var(--grid-desktop-horizontal-spacing) * 2 / 3);
   }
 
   .grid--2-col-desktop .grid__item {
     width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+    max-width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
+}
+
+.grid.grid-full-width {
```

Comment:
Could we use `grid--full-width` instead to match other similar instances like `rich-text--full-width`, `button--full-width`, etc

Reply:
Also do we need the added specificity of `.grid.grid-full-width` here or can we get by without the `.grid`?

Reply:
Mmm right yes 👍

Reply:
Mobile has its own class so I dont think that is an issue. I think we should add `desktop` since it's actually only on desktop

```
@media screen and (min-width: 990px) {
  .grid--full-width .grid__item {
    width: 100%;
    max-width: 100%;
  }
}
```